packages/cli/src/controller/codegen-controller.ts

```diff
@@ -8,33 +8,37 @@ import ejs from 'ejs';
 import {isNonNullType, getNullableType} from 'graphql';
 import {GraphQLFieldMap, GraphQLOutputType} from 'graphql/type/definition';
 import {transformTypes} from './types-mapping';
+
 const templatePath: string = path.resolve(__dirname, '../template/model.ts.ejs');
-const typesPath = `${process.cwd()}/src/types/models`;
 
 // 4. Save the rendered schema
-function makeSchema(className: string, data: string): void {
+export async function makeSchema(className: string, data: string): Promise<void> {
```

Comment:
this abstraction doesn't make sense to me, it is simply wrap of fs.write and once be called once

Reply:
merged into renderTemplate